src/otaclient/ota_core.py

```diff
@@ -962,6 +1091,56 @@ def update(self, request: UpdateRequestV2) -> None:
                 failure_type=e.failure_type,
             )
 
+    def client_update(self, request: ClientUpdateRequestV2) -> None:
+        """
+        NOTE that client update API will not raise any exceptions. The failure information
+            is available via status API.
+        """
+        if os.getenv(cfg.DOWNLOADED_DYNAMIC_OTA_CLIENT):
+            raise NotImplementedError(
+                "multiple dynamic clients downloading is not supported"
+            )
```

Comment:
For downloaded client, allow to wake up multiple clients.
The original otaclient is registered in systemd for restarting. Thus, think it's better to:
- wakeup new client from old client, the status is monitored by old client
- keep alive old client without doing anything(restart is supported by systemd).